packages/main/src/plugin/navigation/navigation-manager.ts

```
@@ -98,6 +105,8 @@ export class NavigationManager {
   @preDestroy()
   dispose(): void {
     this.#disposables.forEach(disposable => disposable.dispose());
+    this.#historyEmitters.forEach(emitter => emitter.dispose());
```

Biome (2.5.3)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)